discovery/triton/triton.go

```diff
@@ -187,6 +192,11 @@ func (d *Discovery) refresh() (tg *targetgroup.Group, err error) {
 	}()
 
 	var endpoint = fmt.Sprintf("https://%s:%d/v%d/discover", d.sdConfig.Endpoint, d.sdConfig.Port, d.sdConfig.Version)
+	if d.sdConfig.Groups != nil {
```

Comment:
`len(d.sdConfig.Groups) > 0` would be more typical

Reply:
Will do